tests/src/readytorun/main.cs

```diff
@@ -49,8 +50,45 @@ public static string OpenClosedDelegateTarget(this string x, string foo)
     }
 }
 
+
+public enum Test
+{
+    Foo,
+    Bar
+}
+
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+public class TestAttribute : Attribute
+{
+    public TestAttribute(Test[] valuesOne, Test[] valuesTwo, Test[] valuesThree, Test[] valuesFour)
+    {
+    }
+}
+
 class Program
 {
+    static void TestCtorAttr()
+    {
+        var type = typeof(Program);
```

Comment:
We'll probably want to migrate these tests to the corefx repo, so we probably want to follow the corefx repo guidelines. I.e. we only use "var" in these situations:

    var x = new Foo();
    var x = (IFoo)something;
    var x = something as IFoo;